### PR TITLE
Setting form - function extraction

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Extract a function for re-usability

Before
----------------------------------------
code less maintainable

After
----------------------------------------
code more maintainable

Technical Details
----------------------------------------
This is a subset of https://github.com/civicrm/civicrm-core/pull/12731 intended to keep the function extraction as clean as possible

Comments
----------------------------------------

### CODE REVIEW

File: CRM/Admin/Form/Setting.php [hunk 1]
```diff
@@ -95,71 +97,11 @@ public function buildQuickForm() {
...
-        }
-        // Migrate to using an array as easier in smart...
-        $descriptions[$setting] = ts($props['description']);
-        $this->assign("{$setting}_description", ts($props['description']));
-        if ($setting == 'max_attachments') {
-          //temp hack @todo fix to get from metadata
-          $this->addRule('max_attachments', ts('Value should be a positive number'), 'positiveInteger');
-        }
-        if ($setting == 'maxFileSize') {
-          //temp hack
```
Comment (colemanw) [inline CRM/Admin/Form/Setting.php]: Not so temp :/

Comment (eileenmcnaughton) [reply CRM/Admin/Form/Setting.php]: lol - it has survived once again

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by mlutfy** [comment]
jenkins, test this please

**Comment by seamuslee001** [comment]
merging as per the tag

**Pull request merged**